doc/src/pair_style.rst

```diff
@@ -67,20 +67,31 @@ If the pair_style command has a cutoff argument, it sets global
 cutoffs for all pairs of atom types.  The distance(s) can be smaller
 or larger than the dimensions of the simulation box.
 
-Typically, the global cutoff value can be overridden for a specific
-pair of atom types by the :doc:`pair_coeff <pair_coeff>` command.  The
-pair style settings (including global cutoffs) can be changed by a
-subsequent pair_style command using the same style.  This will reset
-the cutoffs for all atom type pairs, including those previously set
-explicitly by a :doc:`pair_coeff <pair_coeff>` command.  The exceptions
-to this are that pair_style *table* and *hybrid* settings cannot be
-reset.  A new pair_style command for these styles will wipe out all
-previously specified pair_coeff values.
+In many cases, the global cutoff value can be overridden for a
+specific pair of atom types by the :doc:`pair_coeff <pair_coeff>`
+command.
+
+If a new pair_style command is specified with a new style, all
+previous :doc:`pair_coeff <pair_coeff>` and :doc:`pair_modify
+<pair_modify>` command settings are erased; those commands must be
+re-specified if necessary.
+
+If a new pair_style command is specified with the same style, then
+only the global settings in that command are reset.  Any previous
+doc:`pair_coeff <pair_coeff>` and :doc:`pair_modify <pair_modify>`
+command settings are preserved.  This makes it easy, for example, to
+only change the global cutoff if desired.
```

Comment:
@sjplimp I suggest to add a sentence here, that all *individual* cutoffs from `pair_coeff` commands *will* be reset to the (new) global value by respecifying the pair style. Check out `PairLJCut::settings()` for example.

Reply:
good catch - I made the change